src/Plugin/WebformElement/MineOrganisationsData.php

```diff
@@ -237,7 +237,14 @@ protected function formatTextItemValue(array $element, WebformSubmissionInterfac
 
     foreach ($subElements as $subElement) {
       if (!empty($value[$subElement])) {
-        $lines[$subElement] = $value[$subElement];
+        $title = NestedArray::getValue($element, [
+          '#webform_composite_elements',
+          $subElement,
+          '#title',
+        ]);
+        if (NULL !== $title) {
+          $lines[$subElement] = $title . ': ' . $value[$subElement];
+        }
```

Comment:
This sets `$lines[$subElement]` only when a title is found. Should we add a default value as before?

```suggestion
        $title = NestedArray::getValue($element, [
          '#webform_composite_elements',
          $subElement,
          '#title',
        ]);
        $lines[$subElement] = NULL !== $title ? $title . ': ' . $value[$subElement] : $value[$subElement];
```